### PR TITLE
KFSPTS-20436: Upgrade to the 12-03-2020 version of financials

### PR DESCRIPTION
**Pull request by nkimble**

### CODE REVIEW

**Comment by nkimble** [comment]
These three pull requests need to be merged together when is is appropriate based on current patch validation efforts in KFS-DEV.

PR #67 cu-kfs-upgrades
PR #896 nonprod-sql
PR #1080 cu-kfs

**Review by cah292 — COMMENTED** [review]
Code changes look good, this can be merged once the related PRs are approved and we reach the end of the current sprint.

**Review by cah292 — APPROVED** [review]
Changes look good, unit tests pass, the other PRs are ready, and we're clear to move the patch forward to DEV now. Merging.

**Pull request merged**
